Sources/AppStoreConnectClient/Models/BuildsQuery.swift

```diff
@@ -27,7 +27,63 @@ import Foundation
 /// A structure representing a query for builds with specified sort and fields parameters.
 public struct BuildsQuery {
     /// The sort payload for the query.
-    let sort: Operations.builds_hyphen_get_collection.Input.Query.sortPayload
+    public let sort: [Sort]?
```

Comment:
I think the best solution use empty array as default value. Something like this - `public var sort: [Sort] = []`

Reply:
Changed.